generic/interbench.py

```diff
@@ -49,8 +51,8 @@ def setUp(self):
                     sm_manager.install(pkg)):
                 self.cancel("%s is needed for the test to be run" % pkg)
 
-        disk_free_mb = (disk.freespace(self.teststmpdir) / 1024) / 1024
-        if memory.memtotal()/1024 > disk_free_mb:
+        disk_free_mb = old_div((old_div(disk.freespace(self.teststmpdir), 1024)), 1024)
+        if old_div(memory.memtotal(),1024) > disk_free_mb:
```

Comment:
Here too, It would make travis build successful.

Reply:
@harish-24 noticed it, will take care in the main patch. In this RFC I would be expecting comments on the approach.